.github/workflows/go.yml

```diff
@@ -164,7 +164,7 @@ jobs:
     name: Make Installer (Windows)
     runs-on: windows-2019
     needs: build-windows
-    if: github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/v')
+#   if: github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/v')
```

Comment:
TODO remove this comments.

Reply:
I did it on https://github.com/mackerelio/mackerel-agent/pull/710/commits/bb9f84c0b605873d8f007fbd4748281a91fc0359